lib/pronto/github.rb

```diff
@@ -39,6 +39,8 @@ def create_pull_comment(comment)
     end
 
     def create_pull_request_review(comments)
+      return if comments.empty?
```

Comment:
Another place for adding this could be `GithubPullRequestReviewFormatter#submit_comments`.
Not sure which one should be used.